util/gen-types/Cargo.toml

```diff
@@ -20,7 +20,7 @@ check-data = []
 # Enable the `serialized-size` extension for CKB contract development in `no-std` env
 serialized-size = ["calc-hash"]
 # Enable all in `std` env
-std = ["molecule/std", "ckb-hash/default", "ckb-fixed-hash", "ckb-error", "ckb-occupied-capacity", "numext-fixed-uint"]
```

Comment:
To avoid ambiguity, it is recommended to keeping "ckb-hash/default" or removing “default-features = false“  in `ckb-hash` in [dependencies].

Reply:
I pushed a commit to keep `ckb-hash/default` feature.